src/libraries/System.Numerics.Tensors/src/System/Numerics/Tensors/netcore/TensorPrimitives.netcore.cs

```diff
@@ -13019,6 +13019,105 @@ private static Vector512<T> IsNegative<T>(Vector512<T> vector)
             return Vector512.LessThan(vector, Vector512<T>.Zero);
         }
 
+        /// <summary>poly = C0 + C1*r^3 + C2*r^5 + C3*r^7 + C4*r^9</summary>
+        private static Vector128<T> PolyEvalOdd9<T>(Vector128<T> r, Vector128<T> c0, Vector128<T> c1, Vector128<T> c2, Vector128<T> c3, Vector128<T> c4)
+        {
+            Vector128<T> r2 = r * r;
+            Vector128<T> r4 = r2 * r2;
+
+            Vector128<T> a0 = c2 * r2 + c1;
+            Vector128<T> a1 = a0 * r2 + c0;
+            Vector128<T> a2 = (c3 * r2 + c4 * r4) * r4;
+
+            return r * (a1 + a2);
+        }
+
+        /// <summary>poly = C0 + C1*r^3 + C2*r^5 + C3*r^7 + C4*r^9</summary>
+        private static Vector256<T> PolyEvalOdd9<T>(Vector256<T> r, Vector256<T> c0, Vector256<T> c1, Vector256<T> c2, Vector256<T> c3, Vector256<T> c4)
+        {
+            Vector256<T> r2 = r * r;
+            Vector256<T> r4 = r2 * r2;
+
+            Vector256<T> a0 = c2 * r2 + c1;
+            Vector256<T> a1 = a0 * r2 + c0;
+            Vector256<T> a2 = (c3 * r2 + c4 * r4) * r4;
+
+            return r * (a1 + a2);
+        }
+
+        /// <summary>poly = C0 + C1*r^3 + C2*r^5 + C3*r^7 + C4*r^9</summary>
+        private static Vector512<T> PolyEvalOdd9<T>(Vector512<T> r, Vector512<T> c0, Vector512<T> c1, Vector512<T> c2, Vector512<T> c3, Vector512<T> c4)
+        {
+            Vector512<T> r2 = r * r;
+            Vector512<T> r4 = r2 * r2;
+
+            Vector512<T> a0 = c2 * r2 + c1;
+            Vector512<T> a1 = a0 * r2 + c0;
+            Vector512<T> a2 = (c3 * r2 + c4 * r4) * r4;
+
+            return r * (a1 + a2);
+        }
+
+        /// <summary>poly = C1*r^3 + C2*r^5 + C3*r^7 + C4*r^9 + C5*r^11 + C6*r^13 + C7*r^15 + C8*r^17</summary>
```

Comment:
There appears to be a mismatch between the coefficient argument names and this expansion:
```suggestion
        /// <summary>poly = C1*r^3 + C3*r^5 + C5*r^7 + C7*r^9 + C9*r^11 + C11*r^13 + C13*r^15 + C15*r^17</summary>
```